CHANGELOG_DESKTOP.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [1.18.75](https://github.com/brave/brave-browser/releases/tag/v1.18.75)
+
+ - Added QR code under brave://rewards for iOS devices. ([#13159](https://github.com/brave/brave-browser/issues/13159))
+ - Fixed issue where the New Tab Page was causing persistently higher CPU usage. ([#13194](https://github.com/brave/brave-browser/issues/13194))
```

Comment:
We should add GPU to this sentence as it affected both. Example:

```
- Fixed issue where the New Tab Page was causing persistently higher CPU and GPU usage.
```

Reply:
updated